kube-client/src/client/auth/mod.rs

```diff
@@ -90,6 +90,7 @@ pub(crate) enum Auth {
     Basic(String, SecretString),
     Bearer(SecretString),
     RefreshableToken(RefreshableToken),
+    Certificate(String, String)
```

Comment:
I suppose I should change the second `String` to a `SecretString` to represent the `clientKeyData`?

Reply:
Agreed, good spot. We already  use `secrecy` for this elsewhere!